core/storage/job/gae_models.py

```diff
@@ -89,6 +89,11 @@ def get_new_id(cls, entity_name):
 
     @property
     def is_cancelable(self):
+        """Checks if the job is cancelable.
+
+        Returns:
+            bool. Whether status_code is 'queued' or 'started'.
```

Comment:
Actually, sorry -- can you change this to "Whether the job's status code is ..."? I just realized that there isn't sufficient context to the reader for what status_code is.

Reply:
Done!

Reply:
Thanks @divyadeep1! LGTM. We'll need to wait for @vibhor98 to approve before this can be merged.

Reply:
Okay, thanks. 😄